plugin/web/components/sidebar/Layout/MainLayout.tsx

```diff
@@ -13,25 +13,26 @@ import LayoutHeader from "./LayoutHeader";
 const items: MenuProps["items"] = [
   {
     key: "mapData",
-    icon: <DataBase />,
+    icon: <Icon icon="dataBase" size={24} />,
   },
   {
     key: "mapSetting",
-    icon: <Sliders />,
+    icon: <Icon icon="sliders" size={24} />,
   },
   {
     key: "shareNprint",
-    icon: <Share />,
+    icon: <Icon icon="share" size={24} />,
   },
   {
     key: "about",
-    icon: <Info />,
+    icon: <Icon icon="info" size={24} />,
   },
   {
     key: "template",
-    icon: <Template />,
+    icon: <Icon icon="template" size={24} />,
```

Comment:
If you make the change to the Icon component so that 24 is the default you can remove the size props here (and anywhere else where the icons are 24)

Reply:
done

Reply:
[pr-fix1](https://github.com/eukarya-inc/reearth-plateauview/pull/30/commits/4368dbe8b4bb396983a7cc21a87dc8497771a482)